### PR TITLE
samples: update nPM1300 EK jumpers connection table

### PR DESCRIPTION
**Pull request by jaz1-nordic**
JIRA: NRFX-4660

### CODE REVIEW

**Review by magp-nordic — DISMISSED** [review]

File: samples/timer_watchdog/README.rst [hunk 1]
```diff
@@ -57,21 +57,15 @@ Wiring
...
-
 #. Make the following connections on the chosen DK:
 
    * Set the **SW9** nRF power source switch to **VDD**.
    * Set the **SW10** VEXT → nRF switch to **ON**.
 
+#. Make the following connections on the nPM1300 EK:
+
+   .. include:: /includes/npm1300_ek_connections.txt
+
```
[the remark this answers is not in the document]
Comment (jaz1-nordic) [reply samples/timer_watchdog/README.rst:68]: An oversight ;)

File: samples/vbusin/README.rst [hunk 2]
```diff
@@ -52,21 +52,15 @@ Wiring
...
-
-#. Make the following connections on the DK:
+#. Make the following connections on the chosen DK:
 
    * Set the **SW9** nRF power source switch to **VDD**.
    * Set the **SW10** VEXT → nRF switch to **ON**.
 
+#. Make the following connections on the nPM1300 EK:
+
+   .. include:: /includes/npm1300_ek_connections.txt
```
Comment (magp-nordic) [inline samples/vbusin/README.rst:62]:
```suggestion
   .. include:: /includes/npm1300_ek_connections_with_battery.txt
```
In the description below it is said to connect the battery.

**Comment by wilu-nordic** [comment]
It should be done in a different way. Let's look at how [test_sample](https://github.com/NordicSemiconductor/npmx-zephyr/blob/main/samples/buck/README.rst?plain=1#L125) is done.

**Comment by jaz1-nordic** [comment]
> It should be done in a different way. Let's look at how [test_sample](https://github.com/NordicSemiconductor/npmx-zephyr/blob/main/samples/buck/README.rst?plain=1#L125) is done.

Please specify what exactly you mean.
Do you mean the `replace` directive and replacing only individual lines?
Or should I replace the line `.. include:: /includes/build_and_run.txt` with an additional `replace` directive called, for example `|build_and_run|`?

**Review by wilu-nordic — APPROVED** [review]

**Review by magp-nordic — APPROVED** [review]

**Review by masz-nordic — APPROVED** [review]

**Comment by masz-nordic** [comment]
Already merged in https://github.com/NordicSemiconductor/npmx-zephyr/pull/74

**Pull request closed without merging**